cloudwatch-logs-aggregator/rule/main.tf

```diff
@@ -3,9 +3,10 @@ provider "aws" {
 }
 
 resource "aws_cloudwatch_event_rule" "this" {
-  name                = var.rule_name
-  description         = "created by mackerel-monitoring-modules"
-  is_enabled          = var.is_enabled
+  name        = var.rule_name
+  description = "created by mackerel-monitoring-modules"
+  // TODO we will follow upstream?
+  state               = var.is_enabled
```

Comment:
[should] Former parameter "is_enabled" is boolean but now "state" is DISABLED, ENABLED or ENABLED_WITH_ALL_CLOUDTRAIL_MANAGEMENT_EVENTS. It is confusing to change the type with the same variable, so we recommend creating a new variable.